1.train_models/log_reg_plate_3_3p_cp_renorm_fs_data/nbconverted/train_model.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Training a Logistic Regression Model
+
+# ## Imports
+
+# In[ ]:
+
+
+import sys
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from sklearn.preprocessing import LabelEncoder, MinMaxScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split
+from joblib import dump
+
+
+# ## Find the git root Directory
+
+# In[ ]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
```

Comment:
Given that this is an analysis notebook, is there a point where a `.git` will not exist in someone else's computer? Consider using different approaches to find the root directory. Like using the project name.

If this is not an issue, you can ignore this comment !

Reply:
That's something I thought about as well, I wanted to make it system agnostic. I could have just used a relative path, but then if I move the file, I would have to change this path. If someone clones the repo, than the .git directory should exist. I could use the root folder of the project instead, so the parent of the git directory. I think this would solve the git repo concern, otherwise, the structure and name of the root folder shouldn't really change.